explorer/client/src/utils/api/rpcSetting.ts

```diff
@@ -39,17 +44,21 @@ export const tryGetRpcSetting = (): string | null => {
 };
 
 const ENDPOINTS = {
-    local: 'http://127.0.0.1:5001',
-    devnet: 'https://gateway.devnet.sui.io:443',
+    [Network.Local]: 'http://127.0.0.1:5001',
+    [Network.Devnet]: 'https://gateway.devnet.sui.io:443',
 };
 
-export function getEndpoint(): string {
+export function getEndpoint(network: Network | string): string {
+    //Endpoint has 3 types:
+    // 1) An override value
     const override = tryGetRpcSetting();
     if (override) return override;
```

Comment:
```suggestion
```
We should remove this as well as `tryGetRpcSetting` with this PR since the user can now specify the custom URL through the UI instead of URL encoding it in the URL

Reply:
Yes, the `tryGetRpcLocalStorage` also does not seem to do anything. There's a getItem and a removeItem but, strangely, no setItem.

Reply:
`tryGetRpcSetting` is now removed